app/views/posts/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="form-container border-black">
+  <h1>New Post</h1>
+
+  <%= render "shared/form", post: @post %>
```

Comment:
- [ ] I checked the shared folder and the form partial does not exist thus throwing the error seen below. Please make sure the post creation form renders and a user can create a post using it :+1: 
![image](https://github.com/lucy-sees/Blog/assets/45460192/4e1c152c-b68d-4bcb-9042-d2b3981273da)